python/packages/ai/teams/ai/planner/plan_type.py

```diff
@@ -0,0 +1,9 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+from enum import Enum
+
+
+class PlanType(Enum):
```

Comment:
same string enum comment here

Reply:
see my previous comment, let's discuss in your first enum comment